src/Slim/Middleware/Session.php

```diff
@@ -43,12 +46,16 @@ public function __construct($settings = [])
             'httponly'    => false,
             'name'        => 'slim_session',
             'autorefresh' => false,
+            'handler'     => null,
         ];
         $settings = array_merge($defaults, $settings);
 
         if (is_string($lifetime = $settings['lifetime'])) {
             $settings['lifetime'] = strtotime($lifetime) - time();
         }
+        if (is_string($settings['handler']) && !class_exists($settings['handler'])) {
```

Comment:
Previous `if` does transformation, this one does nothing useful (for me).
PHP takes care of non-existent classes while instantiation, so we can remove this.